sfs/time/drivingfunction.py

```diff
@@ -296,7 +296,18 @@ def matchedz_zpk(s_zero, s_pole, s_gain, fs):
 
 def nfchoa_25d_plane(x0, r0, npw, fs, max_order=None, c=None,
                      s2z=matchedz_zpk):
-    """Virtual plane wave by 2.5-dimensional NFC-HOA.
+    r"""Virtual plane wave by 2.5-dimensional NFC-HOA.
+
+    .. math::
+
+        D(\phi_0, s) =
+        2e^{\frac{s}{c}r_0}
+        \sum_{m=-M}^{M}
+        (-1)^m
+        \Big(\frac{s}{s-\frac{c}{r_0}\sigma_0}\Big)^\mu
+        \prod_{l=1}^{\nu}
+        \frac{s^2}{(s-\frac{c}{r_0}\sigma_l)^2+(\frac{c}{r_0}\omega_l)^2}
+        e^{im(\phi_0 - \phi_\text{pw})}
```

Comment:
You could use `\i` instead of `i` and `\e{...}` instead of `e^{...}` which will result in non italic symbols as proposed original by @fs446.
For an example result see the documentation of `sfs.mono.drivingfunction.nfchoa_25d_plane`

Reply:
Thank you for the review. The math symbols are changed now.